src/components/tooltip/tooltip.scss

```diff
@@ -55,6 +55,7 @@ $m-tooltip--max-height: 280px !default;
 
         &:not(.m--is-link) {
```

Comment:
Retirer la condition `&:not(.m--is-link) {`, car les styles de cette condition seront toujours appliqués puisque la classe `.m--is-link` ne sera jamais ajouter sur la classe `m-tooltip__close-button`. (Voir le fichier tooltip.html pour comprendre)